README.md

```diff
@@ -1,188 +1,194 @@
 
-```bash
-pip install 'ibis-framework[postgres]'
+```python
+>>> ibis.to_sql(g)
+SELECT
+  "t1"."species",
+  "t1"."island",
+  "t1"."count"
+FROM (
+  SELECT
+    "t0"."species",
+    "t0"."island",
+    COUNT(*) AS "count"
+  FROM "penguins" AS "t0"
+  GROUP BY
+    1,
+    2
+) AS "t1"
+ORDER BY
+  "t1"."count" ASC
 ```
 
-Or use `ibis-$BACKEND` where `$BACKEND` is the specific backend you want to use when installing from conda-forge:
+You can mix SQL and Python code:
 
-```bash
-conda install ibis-postgres -c conda-forge
+```python
+>>> t1 = t.sql("SELECT species, island, count(*) AS count FROM penguins GROUP BY 1, 2")
+>>> t1
+┏━━━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━┓
+┃ species   ┃ island    ┃ count ┃
+┡━━━━━━━━━━━╇━━━━━━━━━━━╇━━━━━━━┩
+│ string    │ string    │ int64 │
+├───────────┼───────────┼───────┤
+│ Adelie    │ Torgersen │    52 │
+│ Adelie    │ Biscoe    │    44 │
+│ Adelie    │ Dream     │    56 │
+│ Gentoo    │ Biscoe    │   124 │
+│ Chinstrap │ Dream     │    68 │
+└───────────┴───────────┴───────┘
+>>> t2 = t1.order_by("count")
+>>> t2
+┏━━━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━┓
+┃ species   ┃ island    ┃ count ┃
+┡━━━━━━━━━━━╇━━━━━━━━━━━╇━━━━━━━┩
+│ string    │ string    │ int64 │
+├───────────┼───────────┼───────┤
+│ Adelie    │ Biscoe    │    44 │
+│ Adelie    │ Torgersen │    52 │
+│ Adelie    │ Dream     │    56 │
+│ Chinstrap │ Dream     │    68 │
+│ Gentoo    │ Biscoe    │   124 │
+└───────────┴───────────┴───────┘
 ```
 
-## Getting Started with Ibis
+This allows you to combine the flexibility of Python with the scale and performance of modern SQL.
 
-We provide a number of tutorial and example notebooks in the
-[ibis-examples](https://github.com/ibis-project/ibis-examples). The easiest way
-to try these out is through the online interactive notebook environment
-provided here:
-[![Binder](https://static.mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ibis-project/ibis-examples/main)
+## Backends
 
-You can also get started analyzing any dataset, anywhere with just a few lines
-of Ibis code. Here’s an example of how to use Ibis with a SQLite database.
+Ibis supports 20+ backends:
 
-Download the SQLite database from the `ibis-tutorial-data` GCS (Google Cloud
-Storage) bucket, then connect to it using ibis.
+- [Apache Arrow DataFusion](https://ibis-project.org/backends/datafusion/)
+- [Apache Druid](https://ibis-project.org/backends/druid/)
+- [Apache Flink](https://ibis-project.org/backends/flink)
+- [Apache Impala](https://ibis-project.org/backends/impala/)
+- [Apache PySpark](https://ibis-project.org/backends/pyspark/)
+- [BigQuery](https://ibis-project.org/backends/bigquery/)
+- [ClickHouse](https://ibis-project.org/backends/clickhouse/)
+- [Dask](https://ibis-project.org/backends/dask/)
+- [DuckDB](https://ibis-project.org/backends/duckdb/)
+- [Exasol](https://ibis-project.org/backends/exasol)
+- [HeavyAI](https://github.com/heavyai/ibis-heavyai)
```

Comment:
Can we remove this? HeavyAI 100% **will not** work with any recent version of Ibis.

Reply:
yep!